code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -98,6 +98,37 @@ datum
 			thirst_value = 0.75
 			value = 3 // 1 2
 
+		fooddrink/milk/milk_punch
+			name = "milk punch"
+			id = "milk_punch"
+			fluid_r = 248
+			fluid_g = 200
+			fluid_b = 230
+			transparency = 255
+			taste = "like a mistake"
+			description = "Ugh! It's like a smoothie made of snow cone syrup!"
+			reagent_state = LIQUID
+			thirst_value = 0.3
+			value = 3
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				if(probmult(20))
+					var/mob/living/L = M
+					L.contract_disease(/datum/ailment/disease/food_poisoning, null, null, 1)
+
+		fooddrink/cocktail_fruit_punch
```

Comment:
Personally disagree with reagents for the sake of reagents. Is this all that different from being served a glass that contains various fruit juices etc.?

Reply:
I guess not really; I can take the normal fruit punch one out?

Reply:
Fruit punch is good I think. I wish it were red though!